src/System.Private.ServiceModel/tests/Scenarios/Contract/Data/Contract.Data.Tests.csproj

```diff
@@ -18,6 +18,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="DataContractTests.cs" />
+    <Compile Include="DataContractTests.4.0.0.cs" />
```

Comment:
I would prefer all our projects use the style of <Compile Include="**\*.cs" /> so we don't need to maintain them constantly.  I realize this project was already using the "wrong" style.  The "include all" style is self-maintaining and eliminates the common mistake of adding a test file but forgetting to update the .csproj.

 Not blocking for checking, but consider a follow up clean-up pass.